### ISSUE TITLE
Framework 4.8 Unable to place control

### ISSUE THREAD

**Issue by CArrandale** [VPKSoft/Scintilla.NET#52]
As mentioned, I'm using **Framework 4.8** on **Windows 10** but this is also happening with 4.5.2.

I added Scintilla to a project with framework 4.5 however I need to update the framework to at least 4.5.2 and figured since I'm updating it, might as well update all the way up to modern day (sort of). Everything else updated fine however I'm getting an issue with Scintilla where it broke the form in the designer so I created a new project and tried to add it again and I'm met with an error message: 

```
---------------------------
Microsoft Visual Studio
---------------------------
Failed to create component 'Scintilla'.  The error message follows:

 'System.ComponentModel.Win32Exception (0x80004005): Could not load the Scintilla module at the path 'C:\Users\USERNAME\AppData\Local\Microsoft\VisualStudio\16.0_a4ee1e43\ProjectAssemblies\l7qy_paq01\..\..\build\x86\Scintilla.dll'. ---> System.ComponentModel.Win32Exception (0x80004005): The specified module could not be found

   at ScintillaNET.Scintilla.get_CreateParams()

   at System.Windows.Forms.Control..ctor(Boolean autoInstallSyncContext)

   at System.Windows.Forms.Control..ctor()

   at ScintillaNET.Scintilla..ctor()'
---------------------------
OK   
---------------------------
```

I've tired a couple different versions and still nothing. I might be missing something simple but I feel like I've exhuasted all of my ideas.

***Steps to Reproduce***
1. Create a new VB Framework project.
2. Download Scintilla.NET 5.3.2.4 from NuGet
3. Open Form1.vb in designer
4. Find Scintilla in the toolbox under Scintilla.NET
5. Drag and drop it to the form
6. Met with an error 

Thank  you for your good work and support!

**EDIT:** I should mention that both my main app and test app are both 32bit.

**Comment by VPKSoft** [comment]
Hi,
Seems to be something to do with the way the native Scintilla control is handled in the current version. Try downgrading to v.5.3.2.3 of the NuGet - the change was made in the #50. I'll try to find out why the path gets resolved wrong in the .NET Framework vs .NET.

**Comment by CArrandale** [comment]
That seemed to fix it for **Framwork 4.8**! I didn't test other versions. Thank you very much for your time and effort!
